projects/README.md

```diff
@@ -0,0 +1,14 @@
+## Collect H5 Files
+
+1. Make sure the main hector-slam launch file is up and running.
+1. On a new terminal run `rosrun joy joy_node`
```

Comment:
why not just use the launch file that also starts teleop

Reply:
Because teleop is being started by another thread (collect_h5.py)which is also collecting data. Or we can wrap that into a new launch-file.